src/engine/frameadapterutil.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+// WARNING: These functions are for internal use to assist the migration
+// from sample positions to frame positions in beat related operations.
+// These should only be used to convert between legacy Control Object data
+// which is a double value for sample position and frame position which is
+// an object of the class mixxx::FramePos
+// For example, the sample position is double of the frame position for a
+// stereo track.
```

Comment:
```suggestion
// For example, the sample position is double of the frame position for a
// stereo track.
```
The legacy use of samples for positions assumed interleaved stereo buffers.